crates/astria-cli/src/cli/sequencer.rs

```diff
@@ -10,6 +10,8 @@ use color_eyre::{
     eyre::Context,
 };
 
+const DEFAULT_SEQUENCER_RPC: &str = "https://rpc.sequencer.dusk-1.devnet.astria.org";
```

Comment:
this is already defined as `rollup::DEFAULT_SEQUENCER_RPC`?

Reply:
I can move shared defaults out to it's own module?

I think we ideally avoid sequencer commands being dependent on rollup commands (and vice versa)